auslib/test/admin/views/test_dockerflow.py

```diff
@@ -31,7 +31,9 @@ def testHeartbeatWithException(self):
             cr.side_effect = Exception("kabom!")
             # Because there's no web server between us and the endpoint, we receive
             # the Exception directly instead of a 500 error
-            self.assertRaises(Exception, self.client.get, "/__heartbeat__")
+            # self.assertRaises(Exception, self.client.get, "/__heartbeat__")
+            ret = self.client.get("/__heartbeat__")
+            self.assertEqual(ret.status_code, 500)
             self.assertEqual(cr.call_count, 1)
```

Comment:
Please ensure you test the response body, too, because the bug asks us to ensure that 502s return useful information.